### PR TITLE
Removed chromosome from repo queries

### PR DESCRIPTION
**Pull request by oblodgett**

### CODE REVIEW

**Comment by oblodgett** [comment]
This is linked to: https://github.com/alliance-genome/agr_loader/pull/636

**Comment by oblodgett** [comment]
PS the Chromosome is on the GenomicLocation object so this extra connection is redundant, and this should speed up the Neo4j queries.

File: agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/VariantRepository.java [hunk 1]
```diff
@@ -73,16 +73,4 @@ public List<Allele> getAllelesOfVariant(String variantID) {
 				.collect(Collectors.toList());
 	}
 
-	public GenomeLocation getGenomeLocation(String geneID) {
```
Comment (cmpich) [inline agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/VariantRepository.java:76]: Where was this method being called? I don't see a removal of a calling unit. Maybe it was never used?

Comment (oblodgett) [reply agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/VariantRepository.java:76]: It wasn't being used.

Comment (cmpich) [reply agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/VariantRepository.java:76]: Great. Then it should go out.

**Review by cmpich — APPROVED** [review]
Looks good.

**Pull request merged**